### PR TITLE
Fix bug create account roles with existing policies

### PR DESCRIPTION
**Pull request by oriAdler**
Related: https://issues.redhat.com/browse/SDA-7656

Create managed account roles:
image

Create unmanaged account roles:
image

### CODE REVIEW

**Comment by oriAdler** [comment]
@zgalor @gdbranco PTAL

File: cmd/create/accountroles/cmd.go [hunk 1]
```diff
@@ -398,7 +414,7 @@ func createRoles(r *rosa.Runtime, prefix, permissionsBoundary, accountID, env st
 		}
 
 		filename := fmt.Sprintf("sts_%s_trust_policy", file)
-		policyDetail := aws.GetSTSPolicyDetails(policies, filename)
+		policyDetail := aws.GetManagedPolicyDetails(policies, filename)
```
Comment (zgalor) [inline cmd/create/accountroles/cmd.go:417]: Always get the managed policy? Even if the bool is false?

Comment (oriAdler) [reply cmd/create/accountroles/cmd.go]: Good catch, lousy naming, updated to `GetPolicyDetails`.
`GetPolicyDetails` applies both for unmanaged and managed policies, the function was added for this change https://github.com/openshift/rosa/pull/986/files#diff-97477d2275d7797e234adfc6164c02beabfdbdb0eb516faed787f52d1cfb30c1R521.

The other function `GetManagedPolicyARN` applies only for managed policies.

**Pull request merged**